src/skai/model/train_lib_test.py

```diff
@@ -40,6 +40,7 @@ def _make_serialized_image(size: int, pixel_value: int) -> bytes:
 
 def _make_example(
     example_id: str,
```

Comment:
Let's get rid of this parameter since it's no longer relevant.

Reply:
Thanks for the comments. I'm doing the updates on these.